data/dataset-drug-0-1/dataset.json

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "Drug Image Dataset",
+  "version": "0.1",
+  "name": "drug_data",
+  "dateCreated": "April 13, 2023",
+  "image": "",
+  "description": "Drug Data.",
+  "extra": "Final version of this dataset will be released when associated paper (in review) is published",
+  "publications": [
+  ],
+  "userData": {
+    "isNew": false,
+    "inReview": false,
+    "totalFOVs": 395,
+    "totalTaggedStructures": 1
+  },
+  "featureDefsPath": "feature_defs.json",
+  "featuresDataPath": "cell_feature_analysis.json",
```

Comment:
this needs to match your file name, right now it's `feature_analysis.json`. So you either need to chance this value or change the filename